packages/server-wallet/src/wallet/__test__/integration/update-funding.test.ts

```diff
@@ -186,7 +186,8 @@ it('emits new channel result when the funding event is provided via holdingUpdat
     )
   );
 
-  await w.holdingUpdated({
+  await (w as any).holdingUpdated({
+    // typecasting enables us to acess private method without a TS error
```

Comment:
This doesn't really work, hence the failing test on this branch. 

Perhaps it is a hint that we do actually want these functions to be public?